elide-spring/elide-spring-boot-autoconfigure/pom.xml

```diff
@@ -173,6 +190,12 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <version>${spring.boot.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.vaadin.external.google</groupId>
```

Comment:
How does this conflict?

Reply:
org.json.JSONObject was available from two locations from android-json and everit json schema validator. Everit is used in dynamic config helpers for validation.